platzky/config.py

```diff
@@ -42,7 +42,7 @@ class Config(StrictBaseModel):
     )
     debug: bool = Field(default=False, alias="DEBUG")
     testing: bool = Field(default=False, alias="TESTING")
-    feature_flags: dict[str, bool] = Field(default_factory=dict, alias="FEATURE_FLAGS")
+    feature_flags: t.Optional[dict[str, bool]] = Field(default_factory=dict, alias="FEATURE_FLAGS")
```

Comment:
Client app checkes `PRINT_EXAMPLE`, which is not set in config. What will happen?

Please add test that will check three cases:
- flag set to true
 - flag set to false
 - flag not set
 
 
 THis can be test:
 ```
 def test_if_feature_flag_works():

  config_unset = {some config}
  config_set_true = {some config, FEATURE_FLAGS={PRINT_EXAMPLE:true}}
 config_set_false = {some config, `FEATURE_FLAGS={PRINT_EXAMPLE:false}}}`
 
 create_app = app.blah(config1)
 app.print_example() = lambda _: {
 if app.config.feature_flat: return "printing
 else return "not printing"
 
 assert app_where_unset.print_example == not_printing
 assert app_where_set_false == notrpinting
 assert app_where_set_true == printing
 
 
 ```

Reply:
createed issue https://github.com/Problematy/goodmap/issues/171